tests/src/testassembler.cpp

```diff
@@ -150,6 +154,26 @@ auto SimpleAssemblersTest(const PreBasis& preBasis) {
       }
     }
 
+    constexpr double tol = 1e-8;
+    auto localView       = basis.flat().localView();
+    for (auto& ele : elements(gridView)) {
+      localView.bind(ele);
+      const auto& fe = localView.tree().child(0).finiteElement();
+      std::vector<Dune::FieldVector<double, 2>> nodalPos;
+      Ikarus::obtainLagrangeNodePositions<order>(localView, nodalPos);
+      for (int i = 0; i < fe.size(); i++)
+        if ((std::abs(nodalPos[i][0]) < tol) or (std::abs(nodalPos[i][0] - Lx) < tol)
+            or (std::abs(nodalPos[i][1]) < tol) or (std::abs(nodalPos[i][1] - Ly) < tol))
+          for (auto fixedDirection = 0; fixedDirection < 2; ++fixedDirection) {
+            auto fixIndex = localView.index(localView.tree().child(fixedDirection).localIndex(i));
```

Comment:
The return value of fixIndex is `GlobalIndex`, which is only implicitly convertable to size_t i suppose. See comment above for `fixIthDOF`.

Reply:
You are right. I modified `fixIthDOF`